ui/src/views/network/CreateVpnCustomerGateway.vue

```diff
@@ -317,7 +350,9 @@ export default {
           dpd: values.dpd,
           forceencap: values.forceencap,
           ikepolicy: ikepolicy,
-          esppolicy: esppolicy
+          esppolicy: esppolicy,
+          splitconnections: values.splitconnections,
+          ikeversion: values.ikeversion
```

Comment:
```suggestion
          ikeversion: values.ikeVersions
```

This causes ikeversion to be null at the time of creation of the customer gateway - as it isn't passed, hence defaults to ike
![image](https://user-images.githubusercontent.com/10495417/116030998-74c1a200-a67a-11eb-8473-9fa70b62d88f.png)